src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1415,6 +1416,17 @@ public void setCompactionThroughputMbPerSec(int value)
                     value, oldValue);
     }
 
+    public Map<String, String> getCurrentCompactionThroughput()
+    {
+        HashMap<String, String> result = new LinkedHashMap<>();
+        Meter rate = CompactionManager.instance.getCompactionThroughput();
+        double mb = 1024.0 * 1024.0;
+        result.put("1minute", String.format("%.3f", rate.getOneMinuteRate() / mb));
+        result.put("5minute", String.format("%.3f", rate.getFiveMinuteRate() / mb));
+        result.put("15minute", String.format("%.3f", rate.getFifteenMinuteRate() / mb));
```

Comment:
arent these technically mebibytes, not megabytes? 1024*1024 bytes is one mebibyte, not megabyte, so the unit should be "mib"